### PR TITLE
Rename hasJurisdictionOver. Fixes #637.

### LINKED ISSUE #623: Move gist instances to a separate file?
Consider in conjunction with #305 and #556.

### LINKED ISSUE #637: Rename `hasJurisdictionOver`
This was incorrectly renamed in gist 10 due to a misunderstanding of the intended meaning. Given the definition of `Contract`:

```
gist:Contract
	a owl:Class ;
	owl:equivalentClass [
		a owl:Class ;
		owl:intersectionOf (
			gist:Agreement
			[
				a owl:Restriction ;
				owl:onProperty gist:hasJurisdictionOver ;
				owl:someValuesFrom gist:GovernmentOrganization ;
			]
		) ;
	] ;
.
```
The change to `hasJurisdictionOver` reverses the intended meaning and the restriction is now incorrect. This issue involves a name change; providing a better definition is included in issue #148.

### PR DESCRIPTION
**Pull request by rjyounes**
Corrected the local name and made a quick fix to the definition so that it's intelligible. Further work on the definition is called for in #148.

The history of this is that when local names were updated in gist 10, the meaning of the predicate was misunderstood and so changed. I realized the mistake and have now changed it back.

### CODE REVIEW

File: gistCore.ttl [hunk 1]
```diff
@@ -3137,10 +3137,10 @@ gist:hasIncumbent
 	skos:prefLabel "has incumbent"^^xsd:string ;
 	.
 
-gist:hasJurisdictionOver
+gist:hasJurisdiction
 	a owl:ObjectProperty ;
-	skos:definition "When laws and contracts are meted out"^^xsd:string ;
-	skos:prefLabel "has jurisdiction over"^^xsd:string ;
+	skos:definition "Relates a law, contract, etc., to the system of law or government which has the power, right, or authority to interpret and apply it."^^xsd:string ;
+	skos:prefLabel "has jurisdiction"^^xsd:string ;
```
Comment (uscholdm) [inline gistCore.ttl:3143]: The reason for the confusion is that 'hasJurisdiction' sounds like it means has jurisdiction over. Perhaps a better name would be along the lines of isGovernedByJurisdiction

**Review by uscholdm — CHANGES_REQUESTED** [review]
The reason for the confusion is that 'hasJurisdiction' sounds like it means has jurisdiction over. Perhaps a better name would be along the lines of isGovernedByJurisdiction. Consider an alternative.

File: gistCore.ttl (hunk 1 above)
Comment (rjyounes) [reply gistCore.ttl:3143]: Good idea. I'll use that alternative since it seems pretty good and I can't think of a better one.

Comment (rjyounes) [reply gistCore.ttl]: @uscholdm Changed name to `isUnderJurisdiction`. Not sure if it should be `isUnderJurisdictionOf`.

**Pull request merged**